contracts/src/v0.8/dev/KeeperRegistryLogic.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import "./KeeperRegistryBase.sol";
+
+/**
+ * @notice Logic contract, works in tandem with KeeperRegistry as a proxy
+ */
+contract KeeperRegistryLogic is KeeperRegistryBase {
+  /**
+   * @param link address of the LINK Token
+   * @param linkEthFeed address of the LINK/ETH price feed
+   * @param fastGasFeed address of the Fast Gas price feed
+   */
+  constructor(
+    address link,
+    address linkEthFeed,
+    address fastGasFeed
+  ) KeeperRegistryBase(link, linkEthFeed, fastGasFeed) {}
```

Comment:
Since these values aren't ever actually used, should we set them as zero addresses?

Reply:
Do we need a constructor function on the keeper registry logic contract at all?

Reply:
Yes we need these inside constructor as they are used to set immutable variables which get copied into the code and not storage

Reply:
I think we could just do...
```solidity
  constructor() KeeperRegistryBase(address(0), address(0), address(0)) {}
```

wdyt?